src/App.js

```diff
@@ -1,19 +1,33 @@
 import './App.css';
-import initialCards from './cards-data';
+// import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+// import { useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { useGameContext } from './context/GameContext.js';
 
 function App() {
-  const [deck, setDeck] = useState(initialCards);
-  const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
-  const [playerTwoHand, setPlayerTwoHand] = useState([]);
-  const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
-
+  const {
+    deck,
+    selectedCard,
+    playerOneHand,
+    playerTwoHand,
+    playerThreeHand,
+    setPlayerOneHand,
+    setPlayerTwoHand,
+    setPlayerThreeHand,
+    to,
+    from,
+    setDeck,
+    setSelectedCard,
+  } = useGameContext();
+  // const [deck, setDeck] = useState(initialCards);
```

Comment:
It's a good idea to delete the commented-out code you are no longer using when committing; it keeps the files cleaner and easier to read.